test_tools/log_parser/parser.py

```diff
@@ -141,6 +141,7 @@ def dump_preamble_file(file_name, buf):
     dump_event_batch(buf[PREAMBLE_LENGTH : PREAMBLE_LENGTH + preamble["msg_size"]])
 
 MSG_TYPE_HEADER = 0x55555555
+MSG_TYPE_REWARD_FUNCTION = 0x11111111
```

Comment:
I think this is named `REWARD_FUNCTION` in the joiner and in the parser.. should we change the name there to be consistent?